draft-api/src/main/scala/no/ndla/draftapi/service/ConverterService.scala

```diff
@@ -781,6 +796,18 @@ trait ConverterService {
           .traverse(lang => articleWithNewContent.title.toSeq.map(t => toDomainTitle(api.ArticleTitleDTO(t, t, lang))))
           .flatten
       )
+
+      val updatedDisclaimer = toMergeInto.disclaimer.map { disclaimer =>
+        mergeLanguageFields(
+          disclaimer,
+          maybeLang
+            .traverse(lang =>
+              articleWithNewContent.disclaimer.toSeq.map(d => toDomainDisclaimer(api.DisclaimerDTO(d, lang)))
+            )
+            .flatten
+        )
+      }
```

Comment:
Denne blir vel feil(?)
Vil ikke dette gjøre at man ikke får lagt til nye med mindre det finnes disclaimer fra før av?


Kanskje vi kan gjøre noe sånt (Ikke testet)?

```suggestion
      val updatedDisclaimer = mergeLanguageFields(
        toMergeInto.disclaimer.getOrElse(Seq.empty),
        articleWithNewContent.disclaimer.flatMap { newDisclaimer =>
          maybeLang.map(lang => toDomainDisclaimer(api.DisclaimerDTO(newDisclaimer, lang)))
        }.toSeq
      )
```